### PR TITLE
Add 2021 wg-multitenancy annual report

### PR DESCRIPTION
**Pull request by liggitt**
Copy/paste of wg-multitenancy report

Co-authored-by: Tasha Drew <tasha.drew@gmail.com>

/cc @srampa @tashimi
cc @kubernetes/steering-committee 

/hold for sig and steering review

### CODE REVIEW

**Comment by tashimi** [comment]
/approve 

tenor-194836627

**Review by tashimi — APPROVED** [review]

File: wg-multitenancy/2021-annual-report.md
Comment (nikhita) [inline wg-multitenancy/2021-annual-report.md:19]: This is great! Is there also a rough definition of what completion might look like?

**Review by nikhita — COMMENTED** [review]
Just one question, but LGTM otherwise :tada:

File: wg-multitenancy/2021-annual-report.md
Comment (tashimi) [reply wg-multitenancy/2021-annual-report.md:19]: That's a hard question. Is it when true multi-tenancy capabilities are added to the control plane? Or is it when we don't have projects incubating and let the longer term concerns fall to a SIG, using the large amount of documentation and mindshare the WG has built up over the past few years? I'm open to discussion on which benefits k8s most.

Comment (liggitt) [reply wg-multitenancy/2021-annual-report.md:19]: My take on this is having a sense for:

1. do the current projects have a destination they're tracking toward (e.g. once the benchmarks listed as in-progress are complete, will that project be considered finished?)

2. once the listed projects are finished, does the working group plan to start new projects or consider its mission complete?

Comment (nikhita) [reply wg-multitenancy/2021-annual-report.md:19]: Apologies, should have phrased the question better. What @liggitt said. :)

Comment (tashimi) [reply wg-multitenancy/2021-annual-report.md:19]: Gotcha, yeah. I documented our project plan here: https://docs.google.com/document/d/1U8RQQmTUjxgMZY05HG2f7b3KsB94BhK4Ko6aWbLNXcc/edit

Comment (liggitt) [reply wg-multitenancy/2021-annual-report.md:19]: thanks, added that link in. will let @nikhita take a look and resolve the comment thread.

**Comment by liggitt** [comment]
lgtm (though I bet the bot would be unhappy with me lgtm-ing a PR I opened).

ready for any other steering reviewer feedback

**Review by nikhita — APPROVED** [review]
Thanks @tashimi! 

/lgtm

/hold
for more steering reviews

**Comment by liggitt** [comment]
This lgtm. Emailed the working group for review/comment, and plan to merge next Monday (3/1).

File: wg-multitenancy/2021-annual-report.md
Comment (rjbez17) [inline wg-multitenancy/2021-annual-report.md:40]: Nit: I no longer work at Cray. I'm at Medtronic now :)

Comment (christopherhein) [reply wg-multitenancy/2021-annual-report.md:40]: While I'm not an approver I'm documented in VC `OWNERS` file as a reviewer if helps/makes sense to add Apple to this list.

/cc @parispittman

Comment (liggitt) [reply wg-multitenancy/2021-annual-report.md]: > Nit: I no longer work at Cray. I'm at Medtronic now :)

Updated

> While I'm not an approver I'm documented in VC OWNERS file as a reviewer if helps/makes sense to add Apple to this list.

Thanks, I updated to capture presence of reviewers from other companies

**Comment by liggitt** [comment]
Updated the subprojects companies aspect based on comments in https://github.com/kubernetes/community/compare/ac49ef53094b4f94267ab50671455acf6a509345..04e5513a4f2b5b21843ad0bf8a7d89ddc6194091

/hold cancel

**Comment by tashimi** [comment]
/lgtm 

tenor-234962579

**Pull request merged**
